docs/topics/spiders.rst

```diff
@@ -283,7 +283,19 @@ Spider arguments are passed through the :command:`crawl` command using the
 
     scrapy crawl myspider -a category=electronics
 
-Spiders receive arguments in their constructors::
+Spider arguments are exposed as attributes::
```

Comment:
Hey @demelziraptor !
This is about documenting the feature of spider arguments being exposed as attributes, right? (this line: https://github.com/scrapy/scrapy/blob/master/scrapy/spiders/__init__.py#L30)

From the example it isn't clear what this is trying to show.